send_comments.py

```
@@ -0,0 +1,82 @@
+import requests
+import os
+from dotenv import load_dotenv
+from app.utils.logger import logger
+
+load_dotenv()
+github_token = os.getenv("GITHUB_TOKEN_KEY")
+
+class GitHubCommenter:
+    def __init__(self, owner: str, repo: str, pr_number: int):
+        self.headers = {
+            "Authorization": f"Bearer {github_token}",
+            "Accept": "application/vnd.github+json"
+        }
+        self.owner = owner
+        self.repo = repo
+        self.pr_number = pr_number
+        self.issue_number = pr_number  # PR cũng là Issue
+
+    def send_general_comment(self, message: str):
+        url = f"https://api.github.com/repos/{self.owner}/{self.repo}/issues/{self.issue_number}/comments"
+        data = {
+            "body": message
+        }
+        response = requests.post(url, headers=self.headers, json=data)
```

Calling response.json() without guarding can raise ValueError if the response body is not JSON. Replace with something like:
try:
  content = response.json()
except ValueError:
  content = response.text
Then log response.status_code and content. Also, check response.ok and handle failures (retries or raising exceptions) rather than just printing the result.